SocialCareCaseViewerApi.Tests/V1/Controllers/FormSubmissionControllerTests.cs

```diff
@@ -104,5 +104,54 @@ public void GetSubmissionByIdReturns404WhenACaseIsNotFound()
             response.Should().NotBeNull();
             response?.StatusCode.Should().Be(404);
         }
+
+        [Test]
```

Comment:
Should there be a test for when a request is invalid and a 400 is returned?

Reply:
Yes, good catch :+1: (test added)